### DISCUSSION TITLE
使用了一周，有几个功能建议不知道能否加入

### DISCUSSION THREAD

**Discussion by q943613777** [shion-app/shion#27] [Ideas]
**1、展示组件加入程序图标展示，看程序图标感觉会比看程序名更直观**
![image](https://github.com/shion-app/shion/assets/53282874/c59691c8-90c2-46d4-add1-05bd0e843704)
![image](https://github.com/shion-app/shion/assets/53282874/2f001d13-b7e9-4980-8efe-8138cfeac594)

**2、定时导出数据功能，根据设定的选项在指定目录备份数据，目录地址支持变更**
![1717683301714](https://github.com/shion-app/shion/assets/53282874/5c5592bf-ffe3-47cf-984e-db40e20a202a)

**Comment by hanaTsuk1** [comment]
1. 不考虑（时间线加上图标是因为数据密集，有图标好分辨）
2. 加入开发计划

**Reply by q943613777** [reply]
好的
